tools/c7n_gcp/poetry.lock

```diff
@@ -148,28 +148,34 @@ testing = ["pre-commit"]
 
 [[package]]
 name = "google-api-core"
-version = "2.10.2"
+version = "2.11.0"
 description = "Google API client core library"
 category = "main"
 optional = false
 python-versions = ">=3.7"
 
 [package.dependencies]
-google-auth = ">=1.25.0,<3.0dev"
+google-auth = ">=2.14.1,<3.0dev"
 googleapis-common-protos = ">=1.56.2,<2.0dev"
-grpcio = {version = ">=1.33.2,<2.0dev", optional = true, markers = "extra == \"grpc\""}
-grpcio-status = {version = ">=1.33.2,<2.0dev", optional = true, markers = "extra == \"grpc\""}
+grpcio = [
+    {version = ">=1.33.2,<2.0dev", optional = true, markers = "extra == \"grpc\""},
+    {version = ">=1.49.1,<2.0dev", optional = true, markers = "python_version >= \"3.11\""},
```

Comment:
This 3.11 marker on the new 1.49.1+ grpcio looks like it might be behind the CI failures on this PR :monocle_face:

Reply:
ugh, it isn't a grpc thing, it's a google-api-python-client thing: https://github.com/googleapis/python-api-core/pull/466/files, also partially a poetry thing? maybe